build/configs/rtl8721csm/Make.defs

```diff
@@ -61,9 +61,9 @@ LDSCRIPT = rlx8721d_img2_is.ld
 endif
 
 ifeq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
-LIBSUPXX = ${shell $(CC) $(CXXFLAGS) --print-file-name=libsupc++.a}
+LIBSUPXX = ${shell $(CC) $(CXXFLAGS) --print-file-name=libsupc++_nano.a}
 EXTRA_LIBPATHS += -L "${shell dirname "$(LIBSUPXX)"}"
-EXTRA_LIBS += -lsupc++
+EXTRA_LIBS += -lsupc++_nano
```

Comment:
If this change is only related to exceptions, then let us put it under CONFIG_LIBCXX_EXCEPTION.

Reply:
This sounds good. @Tirtha-Samsung please verify the booting of the board on cxxtest config too with the changes. I haven't tested libsupc++_nano with flat build before.

Reply:
I will verify it

Reply:
> This sounds good. @Tirtha-Samsung please verify the booting of the board on cxxtest config too with the changes. I haven't tested libsupc++_nano with flat build before.

@Tirtha-Samsung Is this verified on the flat?

Reply:
Yes I verified cxxtest->helloxx, it is booting fine